app/assets/javascripts/users/profile_edit.js

```diff
@@ -0,0 +1,12 @@
+(function(){
+  $(document).ready(function(){
+    setDonationBars();
+  });
+  function setDonationBars(){
+    $(".donations-bars > .donation-bar-container").each(function(index){
+      _that = $(this);
+      var container_width = $(".donations-bars").width();
+      _that.css("width", _that.data("percent") * 1.0 * 75 / 100 + "%");
```

Comment:
This works, but it's not a good practice to mix arithmetic logic mixed with string concatenation. We should try to abstract away to a different method, to where it looks something like:
```
_that.css("width", computeCssPercent(_that.data) + "%");
```

Reply:
Yeah, good point. I can take care of that.